test/Strings/rlexe.xml

```diff
@@ -230,4 +230,11 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>unicode_toUpperCase_toLowerCase.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>exclude_win7</tags>
```

Comment:
I'm curious what the reason to exclude windows 7 is? Does the WinAPI you are using not exist in 7? Does this bug need to be reopened against the linux branch if chakra is using windows 8+ specific APIs to fix this bug?